.github/workflows/pre-release.yml

```diff
@@ -76,9 +88,23 @@ jobs:
           path: ${{steps.vars.outputs.nwakutools}}
           retention-days: 2
 
+  build-docker-image:
+    needs: tag-name
+    uses: waku-org/nwaku/.github/workflows/container-image.yml@master
+    with:
+      image_tag: ${{ needs.tag-name.outputs.tag }}
+    secrets: inherit
+
+  js-waku:
+    needs: build-docker-image
+    uses: waku-org/js-waku/.github/workflows/ci.yml@master
+    with:
+      nim_wakunode_image: ${{ needs.build-docker-image.outputs.image }}
+      caller: nwaku
```

Comment:
For me to understand: what does that do?
Thanks in advance!

Reply:
Yes, `uses` at `job` level specifies the workflow YAML file to use for the job to execute (can be local to the repo or external -as seen here), then `with` specifes input parameters for the called workflow - the PR/pre-release image and then also specify the call comes from `nwaku` so that we can limit the jobs in js-waku CI that get executed (e.g. we do not want to do release:) )